src/main/msp/msp.c

```diff
@@ -1209,11 +1209,34 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
                 rpm = (int)getDshotTelemetry(i) * 100 * 2 / motorConfig()->motorPoleCount;
                 rpmDataAvailable = true;
                 invalidPct = 10000; // 100.00%
+
 #ifdef USE_DSHOT_TELEMETRY_STATS
                 if (isDshotMotorTelemetryActive(i)) {
                     invalidPct = getDshotTelemetryMotorInvalidPercent(i);
                 }
 #endif
+
+                // Provide extended dshot telemetry
+                if ((dshotTelemetryState.motorState[i].telemetryTypes & DSHOT_EXTENDED_TELEMETRY_MASK) != 0)
+                {
```

Comment:
All non-function statement blocks (i.e. if, switch, for, as well as any others) shall have the opening brace last on the same line, with the following statement on the next line.

See: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#non-function-statement-blocks

Reply:
Fixed into the squash